<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1296</width>
    <height>778</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayoutCollectorAllLeft">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutContentUpdateLeft">
       <item>
        <widget class="QPushButton" name="pushButtonUpdateListLeft">
         <property name="font">
          <font>
           <family>Noto Sans Gothic</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::NoContextMenu</enum>
         </property>
         <property name="toolTip">
          <string>Update the list of collectors based on the region/credentials entered above. </string>
         </property>
         <property name="text">
          <string>Update</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutCollectorLabelLeft">
       <item>
        <widget class="QLabel" name="labelCollectorsLeft">
         <property name="font">
          <font>
           <family>Noto Sans Gothic</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Collectors:</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditCollectorSearchLeft">
         <property name="toolTip">
          <string>Filter the list of collectors based on the string that you type here. This is case sensistive. </string>
         </property>
         <property name="placeholderText">
          <string>Search</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="listWidgetCollectorsLeft">
       <property name="font">
        <font>
         <family>Waree</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>false</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutCollectorButtonsLeft">
       <item>
        <widget class="QPushButton" name="pushButtonBackupCollectorLeft">
         <property name="font">
          <font>
           <family>Noto Sans Gothic</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Backup the selected collector(s) to JSON file. There is currently no collector restore, but you can restore source(s) to an existing collector based on this backup. </string>
         </property>
         <property name="text">
          <string>Backup Collector(s)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonDeleteCollectorLeft">
         <property name="font">
          <font>
           <family>Noto Sans Gothic</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Delete the selected collector(s). Be careful. Deleting the wrong one(s) could be very bad. </string>
         </property>
         <property name="text">
          <string>Delete Collector(s)</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutCollectorSourceLabelLeft">
       <item>
        <widget class="QLabel" name="labelSourcesLeft">
         <property name="font">
          <font>
           <family>Noto Sans Gothic</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Sources:</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="listWidgetSourcesLeft">
       <property name="font">
        <font>
         <family>Waree</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>false</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutCollectorSourceButtonsLeft">
       <item>
        <widget class="QPushButton" name="pushButtonDeleteSourcesLeft">
         <property name="font">
          <font>
           <family>Noto Sans Gothic</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Delete the selected source(s). Be careful. Deleting the wrong one(s) could be very bad. </string>
         </property>
         <property name="text">
          <string>Delete Source(s)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonRestoreSourcesLeft">
         <property name="font">
          <font>
           <family>Noto Sans Gothic</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Restore source(s) from JSON file, such as a collector backup. Does not create the collector itself though.  </string>
         </property>
         <property name="text">
          <string>Restore Source(s)</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayoutCollectorCentralButtons">
     <item>
      <spacer name="verticalSpacer_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelCollectorLegend">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Noto Sans Gothic</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Collector&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;Legend:&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;------------&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;Bold: Hosted&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:400;&quot;&gt;Normal: Installed&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:400; font-style:italic;&quot;&gt;Italics: Disconnected&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_6">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCopySourcesLeftToRight">
       <property name="font">
        <font>
         <family>Noto Sans Gothic</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="toolTip">
        <string>Copy the selected source(s) from the left to the selected collector(s) on the right. Use this to mass deploy sources to many collectors. </string>
       </property>
       <property name="text">
        <string>Copy -&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCopySourcesRightToLeft">
       <property name="font">
        <font>
         <family>Noto Sans Gothic</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="toolTip">
        <string>Copy the selected source(s) from the right to the selected collector(s) on the left. Use this to mass deploy sources to many collectors. </string>
       </property>
       <property name="text">
        <string>&lt;- Copy</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_7">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayoutCollectorAllRight">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutContentUpdateRight">
       <item>
        <widget class="QPushButton" name="pushButtonUpdateListRight">
         <property name="font">
          <font>
           <family>Noto Sans Gothic</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::NoContextMenu</enum>
         </property>
         <property name="toolTip">
          <string>Update the list of collectors based on the region/credentials entered above. </string>
         </property>
         <property name="text">
          <string>Update</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutCollectorLabelRIght">
       <item>
        <widget class="QLabel" name="labelCollectorsRight">
         <property name="font">
          <font>
           <family>Noto Sans Gothic</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Collectors:</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditCollectorSearchRight">
         <property name="toolTip">
          <string>Filter the list of collectors based on the string that you type here. This is case sensistive. </string>
         </property>
         <property name="placeholderText">
          <string>Search</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="listWidgetCollectorsRight">
       <property name="font">
        <font>
         <family>Waree</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>false</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutCollectorButtonsRight">
       <item>
        <widget class="QPushButton" name="pushButtonBackupCollectorRight">
         <property name="font">
          <font>
           <family>Noto Sans Gothic</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Backup the selected collector(s) to JSON file. There is currently no collector restore, but you can restore source(s) to an existing collector based on this backup. </string>
         </property>
         <property name="text">
          <string>Backup Collector(s)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonDeleteCollectorRight">
         <property name="font">
          <font>
           <family>Noto Sans Gothic</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Delete the selected collector(s). Be careful. Deleting the wrong one(s) could be very bad. </string>
         </property>
         <property name="text">
          <string>Delete Collector(s)</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_8">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutCollectorSourceLabelRight">
       <item>
        <widget class="QLabel" name="labelSourcesRight">
         <property name="font">
          <font>
           <family>Noto Sans Gothic</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Sources:</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_9">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="listWidgetSourcesRight">
       <property name="font">
        <font>
         <family>Waree</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>false</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutCollectorSourceButtonsRight">
       <item>
        <widget class="QPushButton" name="pushButtonDeleteSourcesRight">
         <property name="font">
          <font>
           <family>Noto Sans Gothic</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Delete the selected source(s). Be careful. Deleting the wrong one(s) could be very bad. </string>
         </property>
         <property name="text">
          <string>Delete Source(s)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonRestoreSourcesRight">
         <property name="font">
          <font>
           <family>Noto Sans Gothic</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Restore source(s) from JSON file, such as a collector backup. Does not create the collector itself though.  </string>
         </property>
         <property name="text">
          <string>Restore Source(s)</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_10">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
